src/Microsoft.DotNet.ProjectModel/Compilation/LibraryExporter.cs

```diff
@@ -184,22 +190,24 @@ private LibraryExport ExportProject(ProjectDescription project)
                     Path.GetFullPath(Path.Combine(project.Project.ProjectDirectory, assemblyPath)));
 
                 compileAssemblies.Add(compileAsset);
-                runtimeAssets.Add(pdbPath);
+                runtimeAssets.Add(new LibraryAsset(pdbPath, Path.GetFileName(pdbPath), pdbPath));
```

Comment:
It's common structure that needs name, relative path, and resolved path, changed it to put `Path.GetFileName` as a `name` parameter